providers/edge/src/airflow/providers/edge/cli/edge_command.py

```diff
@@ -470,6 +477,12 @@ def stop(args):
     ("-H", "--edge-hostname"),
     help="Set the hostname of worker if you have multiple workers on a single machine",
 )
+ARG_WAIT = Arg(
+    ("-w", "--wait"),
+    default=False,
+    help="Wait until edge worker is shot down.",
```

Comment:
Typo. That made me a laugh Monday morning.
```suggestion
    help="Wait until edge worker is shut down.",
```